operate/services/service.py

```diff
@@ -972,6 +976,35 @@ def deployment(self) -> Deployment:
             self._deployment = Deployment.new(path=self.path)
         return t.cast(Deployment, self._deployment)
 
+    @property
+    def package_absolute_path(self) -> Path:
+        """Get the package_absolute_path."""
+        self._ensure_package_exists()
+        package_absolute_path = self.path / self.package_path
+        return package_absolute_path
+
+    def _ensure_package_exists(self) -> None:
+        package_absolute_path = self.path / self.package_path
+        if (
+            not package_absolute_path.exists()
+            or not (package_absolute_path / "service.yaml").exists()
```

Comment:
@OjusWiZard  Should we also test here that `service.yaml` is not corrupted? would that overload too much this call?

Reply:
I think it can be avoided for now to keep it fast. Hashing the file for every `package_absolute_path` call will be expensive.

Reply:
Agree.